frontend/src/app/modules/user-profile/user-details/user-details/user-details.component.html

```diff
@@ -6,200 +6,172 @@
         <div class="avatar-wrp">
             <app-avatar [size]="103"></app-avatar>
             <div class="edit-avatar-icon">
-                <img src="/assets/user-profile//edit-icon.svg" alt="edit" />
+                <img src="/assets/user-profile//edit-icon.svg" alt="edit"/>
             </div>
             <div class="vertical-line"></div>
         </div>
         <div class="avatar-list">
             <div class="avatar-suggestion-wrp">
-                <img src="/assets//user-profile/boy.svg" alt="" />
+                <img src="/assets//user-profile/boy.svg" alt=""/>
             </div>
             <div class="avatar-suggestion-wrp">
-                <img src="/assets//user-profile/man-bald.svg" alt="" />
+                <img src="/assets//user-profile/man-bald.svg" alt=""/>
             </div>
             <div class="avatar-suggestion-wrp">
-                <img src="/assets//user-profile/man-beard.svg" alt="" />
+                <img src="/assets//user-profile/man-beard.svg" alt=""/>
             </div>
             <div class="avatar-suggestion-wrp">
-                <img src="/assets//user-profile/man-curly.svg" alt="" />
+                <img src="/assets//user-profile/man-curly.svg" alt=""/>
             </div>
             <div class="avatar-suggestion-wrp">
-                <img src="/assets//user-profile/man-elf.svg" alt="" />
+                <img src="/assets//user-profile/man-elf.svg" alt=""/>
             </div>
             <div class="avatar-suggestion-wrp">
-                <img src="/assets//user-profile/man-in-tuxedo.svg" alt="" />
+                <img src="/assets//user-profile/man-in-tuxedo.svg" alt=""/>
             </div>
             <div class="avatar-suggestion-wrp">
-                <img src="/assets//user-profile/man-red-hair.svg" alt="" />
+                <img src="/assets//user-profile/man-red-hair.svg" alt=""/>
             </div>
             <div class="avatar-suggestion-wrp">
-                <img src="/assets//user-profile/man-technologist.svg" alt="" />
+                <img src="/assets//user-profile/man-technologist.svg" alt=""/>
             </div>
             <div class="avatar-suggestion-wrp">
-                <img src="/assets//user-profile/man-superhero.svg" alt="" />
+                <img src="/assets//user-profile/man-superhero.svg" alt=""/>
             </div>
             <div class="avatar-suggestion-wrp">
-                <img src="/assets//user-profile/man-white-hair.svg" alt="" />
+                <img src="/assets//user-profile/man-white-hair.svg" alt=""/>
```

Comment:
Any reason why we have one slash somewhere and two at other places?

Reply:
It's not my changes only formating was changed by WebStorm. What do you mean, pls clarify?

Reply:
`<img src="/assets**//**user-profile/man-white-hair.svg" alt=""/>`

Reply:
Check slashes
/assets//user-profile/man-white-hair.svg
/assets/user-profile//edit-icon.svg

Please, remove this unneeded ones